webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9481,6 +9481,14 @@ const firstVisibleMsg = {
                     }
                 }
 
+                if (
+                    msg &&
```

Comment:
what is the purpose of Kommunicate.visibleMessage ? is that meant for this purpose and check needs to go there?

Reply:
I have added this in addMessage. Not sure why it is showing here. This variable is defined in start of code